.github/workflows/ci.yml

```diff
@@ -172,93 +172,3 @@ jobs:
         with:
           name: coverage
           path: coverage
-
-  build-skip-check:
-    runs-on: ubuntu-22.04
-    outputs:
-      skip: ${{ steps.skip-check.outputs.skip }}
-    steps:
-      - name: Skip?
-        id: skip-check
-        run: |
-          if [[ "${{ vars.DOCKER_USER }}" == '' ]]; then
-            echo 'Docker user is empty. Skipping build+push'
-            echo skip=true >> "$GITHUB_OUTPUT"
-          elif [[ "${{ secrets.DOCKER_PASS }}" == '' ]]; then
-            echo 'Docker password is empty. Skipping build+push'
-            echo skip=true >> "$GITHUB_OUTPUT"
-          elif [[ "${{ github.ref_name }}" != 'master' ]]; then
```

Comment:
@eradman instead of moving everything to a separate file, its enough to change `github.ref_name` variable. need to check documentation

Reply:
For sure, these actions can be combined in one yaml file, but I found these rules to be easier to read in a separate file